felix/fv/bpf_config_map_size.go

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2021 Tigera, Inc. All rights reserved.
```

Comment:
the file name should end with `_test.go` What about `bpf_map_resize/changed_test.go` ? We will also add conntrack migrations here etc. What about merging it with bpf_attach_test into a `bpf_restart_test` or `bpf_config_changed_test` ?

Reply:
I feel its better to have these tests in separate files. Will rename this file ```bpf_map_resize_test.go```